notifications/tasks.py

```diff
@@ -83,10 +110,40 @@ def render_email_html_message(recipient, message, profile_url, profile_type):
     - profile_url
     - profile_type
     """
+    link_to_profile = f"<p><a href={profile_url}>View {profile_type}'s profile</a></p>"
     html_message = f'''
     <p>Hello, {recipient.get_full_name()}</p>
     <p>{message}</p>
-    <p><a href={profile_url}>View {profile_type}'s profile</a></p>
+    {link_to_profile if include_link else ''}
     <p>Thank you for choosing Forum!</p>
     '''
     return html_message
+
+
+@shared_task
+def set_initial_notification_settings(instance_id, role_name):
+    """set initial notification settings for new profile
+    
+    Fields:
+    - instance_id
+    - role_name
+    """
+    instance = None
+    try:
+        role = Role.objects.get(name=role_name)
+        if role_name == 'Startup':
+            instance = StartUpProfile.objects.get(id=instance_id)
+        elif role_name == 'Investor':
+            instance = InvestorProfile.objects.get(id=instance_id)
+    except (Role.DoesNotExist, InvestorProfile.DoesNotExist, StartUpProfile.DoesNotExist) as e:
+        logger.error(f'{e} Created {role_name} instance not found: {instance_id}')
+
+    allowed_notifications = RolesNotifications.objects.filter(role=role)
+
+    if instance:
+        for notification in allowed_notifications:
+            NotificationPreferences.objects.get_or_create(
+                user=instance.get_user(),
+                role=role,
+                notification_type=notification.notification_type
+            )
```

Comment:
If we want to add logging for this file, then we should add it using different levels, such as DEBUG, INFO, etc.